package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "meteor-coverage",
+  "version": "0.9.1",
+  "description": "add istanbul code instrumentation to meteor code so we can have coverage analysis",
+  "main": "package.js",
+  "scripts": {
+    "lint": "eslint .",
+    "test": "COVERAGE=1 COVERAGE_APP_FOLDER=./ COVERAGE_VERBOSE=1 meteor test-packages --once --driver-package dispatch:mocha-phantomjs ./"
```

Comment:
full path required for COVERAGE_APP_FOLDER ! I don't think such path will work all the time in every case.